src/platform_impl/linux/wayland/seat/keyboard/handlers.rs

```diff
@@ -0,0 +1,151 @@
+//! Handling of various keyboard events.
+
+use sctk::reexports::client::protocol::wl_keyboard::KeyState;
+
+use sctk::seat::keyboard::Event as KeyboardEvent;
+
+use crate::event::{ElementState, KeyboardInput, ModifiersState, WindowEvent};
+use crate::platform_impl::wayland::event_loop::WinitState;
+use crate::platform_impl::wayland::{self, DeviceId};
+
+use super::keymap;
+use super::KeyboardInner;
+
+#[inline]
```

Comment:
Well, initially I just decided to move handler to a different file, since it was approach quite big, but since handler is used only in one place I want to inline it to be more efficient.

Reply:
That seems fair, though unless you have benchmarked it there is no evidence that it is indeed more efficient, right?

Especially with big methods having it inlined can also make things slower, but it does indeed provide value for additional optimizations within the function, so it makes sense.